<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditorPreferencesDialog</class>
 <widget class="QWidget" name="MapEditorPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>198</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map editor preferences</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Movement speeds</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="objectRotationLbl">
      <property name="text">
       <string>Object rotation</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="objectRotationTxt">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="objectMoveLbl">
      <property name="text">
       <string>Object move</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="objectMoveTxt">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="cameraRotationLbl">
      <property name="text">
       <string>Camera rotation</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="cameraRotationTxt">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="cameraZoomLbl">
      <property name="text">
       <string>Camera zoom</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="cameraZoomTxt">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
